Dashboard: initial styling and structure tweaks

jwmclaren opened the pull request:
Improved CSS styling, as well as modifying page structure to minorly improve things.

jwmclaren committed:
Dashboard: initial styling and structure tweaks

Improved CSS styling, as well as modifying page structure to minorly improve things.

obijywk commented on public/dashboard.css:
```diff
@@ -1,33 +1,43 @@
-  width: 100%;
-  height: 100%;
+html {
+  --dashboard-radius: 8px;
```
I think we should try to avoid using any px units... we don't know the resolutions of the screens we'll be running on, and they may change drastically year-to-year.

obijywk commented on public/dashboard.css:
```diff
@@ -38,123 +48,149 @@ div.stat-box {
 
 span.stat {
-  font-size: 5vh;
+  font-family: 'Roboto Mono', monospace;
```
why monospace for these? with numbers in the tens and hundreds I think it's gonna get squishy, and the spaces around the slashes end up taking up too much space.

obijywk commented on public/dashboard.css:
```diff
@@ -38,123 +48,149 @@ div.stat-box {
+  border: 2px solid #e1e1e1;
+}
+
+div.pinned {
```
this pinned class is not used yet? do we need some code support to make this possible, or can we figure out which puzzles to apply this style to via CSS selectors (e.g. searching the div.puzzle descendant elements for an element with class pin or something)?

obijywk replied:
although maybe now this is fixed, i was reviewing before you updated

obijywk commented on public/dashboard.css:
```diff
@@ -38,123 +48,149 @@ div.stat-box {
-  font-size: 3vh;
-  white-space: nowrap;
-  overflow: hidden;
+div.puzzle-header{
```
```suggestion
div.puzzle-header {
```

obijywk commented on views/dashboard.hbs:
```diff
@@ -11,7 +11,6 @@
   </head>
   <body>
      <div class="dashboard">
```
there's some problem here where indentation isn't lining up, I can fix it later

jwmclaren committed:
List twelve puzzles on dashboard, and highlight pins

obijywk commented on public/dashboard.css:
```diff
@@ -38,123 +48,149 @@ div.stat-box {
+  overflow: hidden;
 }
 
 div.puzzle-people img {
```
I do like the profile pics, but we should figure out some way to cap the number of these we show (and maybe do like a +N thing for the number we dropped, although that'll probably be more complicated)

jwmclaren committed:
Merge branch 'master' into dashboard

jwmclaren committed:
Added headings to sections

obijywk committed:
on the dashboard, cap the number of user profile images we show for each puzzle

jwmclaren committed:
Improved capped solver display.

obijywk committed:
Merge pull request #51 from obijywk/dashboard-capped-users

on the dashboard, cap the number of user profile images we show for each puzzle

jwmclaren committed:
Up cappedUser count to 10

obijywk committed:
Merge pull request #52 from obijywk/dashboard-capped-users

Up cappedUser count to 10

obijywk committed:
Merge branch 'master' into dashboard

jwmclaren committed:
Updates to dashboard CSS

* Moved tags to allow for more tags before content collision issues occur.
* Tightened spacings to maintain readability

obijywk committed:
Many dashboard changes:

 - change all pixel-based units to viewport-size-relative units
 - remove stats title and reduce height of stats box, to make more room for events
 - add more detail to section titles (why not, we have the space)
 - reduce line spacing / padding within notices and events sections
 - replace long links in notices and events sections with <long link>
 - order notices section by most recent pins first
 - puzzle detail sizing and spacing tweaks
 - fix dashboard HTML indentation

obijywk committed:
Dashboard: add more detail to section titles and add a little more vertical spacing within puzzle details
